lambda/main_test.go

```diff
@@ -53,3 +53,43 @@ func TestMain(t *testing.T) {
 	})
 	assert.NoError(t, err)
 }
+
+
+func TestMainWithImageArch(t *testing.T) {
```

Comment:
This test case doesn't appear to test any of the code, instead it tests the library we are using.

Reply:
Updated